### ISSUE TITLE
Pagination component loses focus when start or end of pagination is reached

### ISSUE THREAD

**Issue by straker** [dequelabs/cauldron#703]
The bowser focus jumps back to the `body` element when the user is using the keyboard to navigate the pagination and reaches either the start or the end (using any of the buttons). Focus should remain on the button when the start / end of the pagination is reached.

**Comment by ryomtoob** [comment]
This component uses a combination of `<IconButton>` and `<TooltipTabstop>`. This is so that the disabled pagination buttons can still be focused and display a tooltip indicating direction. When the start or end is reached, the `<IconButton>` is swapped for a `<TooltipTabstop>` which cause focus to be lost. To fix this, focus should be placed on the appropriate `<TooltipTabstop>` once the user reaches the start or end of pagination.

**Comment by joanjeremiah** [comment]
I would like to solve this issue. Kindly assign this task to me.

**Comment by anastasialanz** [comment]
@joanjeremiah I assigned this one to you. Thanks!

**Comment by VaniChinnam** [comment]
Verified in Cauldron - https://cauldron.dequelabs.com/components/Pagination
The focus order is proper. On activating any of the pagination links, the focus stays on that button itself.
